### PR TITLE
Added sample form

### PR DESCRIPTION
**Pull request by isomer04**
Screenshot_512

### CODE REVIEW

**Comment by mjbramich** [comment]
@isomer04 One question you've added a dependency to work with forms, should we add this or not? @IntelliJinceTech @AitchGrasso @Michelle-932

**Comment by isomer04** [comment]
> @isomer04 One question you've added a dependency to work with forms, should we add this or not? @IntelliJinceTech @AitchGrasso @Michelle-932

It will be useful for form validation. I can remove it if you guys don't want it

**Comment by IntelliJinceTech** [comment]
Let's Review this PR together since I don't really know how it works and looks given the current commits that we have on main. @isomer04 @mjbramich @Michelle-932 @AitchGrasso

File: client/src/components/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+const Form = () => {
+  const { register, handleSubmit, reset } = useForm();
+
+  const onSubmit = (data) => {
+    // You can perform additional logic here, like sending the form data to a server
```
Comment (IntelliJinceTech) [inline client/src/components/Form.jsx:8]: Submits usually need something to prevent the re-render. `preventDefault()`?

**Review by IntelliJinceTech — APPROVED** [review]
Good start. I think we can build off this. I'm going to approve of this PR but I would be remiss if I didn't allow more people to review this as well before we actually merge it. It also speaks to my inexperience with the `react-hook-form` module. 

I did make a comment on the `onSubmit` form handler. 

Please make sure that if you haven't already - run es lint.

**Comment by mjbramich** [comment]
Similar to Chris, inexperienced with the React-hook-form module, so i will need to familiarise with it. 
Just check for eslint errors too.

**Pull request closed without merging**